vaadin-date-time-picker-flow-demo/src/main/java/com/vaadin/flow/component/datetimepicker/demo/DateTimePickerView.java

```diff
@@ -118,6 +119,18 @@ private void finnishDateTimePicker() {
         Locale localeFI = new Locale("fi");
         dateTimePicker.setLocale(localeFI);
 
+        dateTimePicker.setDatePickerI18n(new DatePicker.DatePickerI18n().setWeek("viikko")
+                .setCalendar("kalenteri").setClear("tyhjennä")
+                .setToday("tänään").setCancel("peruuta").setFirstDayOfWeek(1)
+                .setMonthNames(Arrays.asList("tammiku", "helmikuu", "maaliskuu",
```

Comment:
First string should be "tammikuu" (missing last letter)

Reply:
Updated.